scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/solver/PrioritySolver.java

```diff
@@ -467,6 +467,13 @@ private void satisfyCompositeGoal(CompositeAndGoal goal) {
       if (evaluation.forGoal(subgoal).getScore() == 0) {
         logger.info("AND goal " + goal.getName() + ": subgoal " + subgoal.getName() + " has been satisfied, moving on to next subgoal");
         nbGoalSatisfied++;
+      } else {
+        logger.info("AND goal " + goal.getName() + ": subgoal " + subgoal.getName() + " has NOT been satisfied");
+        if(goal.shouldRollbackIfUnsatisfied()){
+          logger.info("AND goal " + goal.getName() + ": stopping goal satisfaction after first failure, use shouldRollbackIfUnsatisfied() on AND goal to maximize satisfaction instead of early termination");
```

Comment:
I meant using the function shouldRollbackIfUnsatisfied(boolean). I have replaced the message with 
`          logger.info("AND goal " + goal.getName() + ": stopping goal satisfaction after first failure, remove shouldRollbackIfUnsatisfied(true) on AND goal to maximize satisfaction instead of early termination");`